manifests/storage.pp

```diff
@@ -43,10 +43,9 @@
 ) inherits bacula {
```

Comment:
Is the inherits enough to bring into scope the `bacula::db_type` that's used below?  I remember some waffling back and forth on this in past versions of puppet, but can't recall the details.

Reply:
I guess so: according to [this old doc](https://puppet.com/docs/puppet/4.10/lang_classes.html#inheritance):

« Inheritance causes three things to happen:

1. When a derived class is declared, its base class is automatically declared first (if it wasn’t already declared elsewhere). »

I could not find the corresponding documentation in recent documentation (because of reorganization I hope), but FYI I have deployed this branch on one of my production sites, and ensured the change where noop.